client/auth/base/api.js

```diff
@@ -62,7 +62,7 @@ define([
       client_id: this._clientId,
       state: config.state,
       scope: config.scope,
-      redirect_uri: config.redirect_uri
+      redirect_uri: config.redirectUri
```

Comment:
the `redirect_uri` on the left must remain the same - this is passed to the oauth server.